cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -399,8 +399,11 @@ export class CanvasViewImpl implements CanvasView, Listener {
                 this.svgTexts[state.clientID].remove();
             }
 
-            this.svgShapes[state.clientID].off('click.canvas');
-            this.svgShapes[state.clientID].remove();
+            const shape = this.svgShapes[state.clientID];
```

Comment:
Here and in other places in this file you check if ``points`` or ``shape`` exist before using it, right?
But why do you need it if canvas doesn't know about tags? (it doesn't draw them anyhow)